# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../system-config-printer/replace_rhpl_with_gettext.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -urN system-config-printer-0.9.93.orig/cupshelpers.py system-config-printer-0.9.93/cupshelpers.py
--- system-config-printer-0.9.93.orig/cupshelpers.py	2008-05-23 21:57:31.000000000 +1000
+++ system-config-printer-0.9.93/cupshelpers.py	2008-09-21 21:55:17.000000000 +1000
@@ -19,7 +19,7 @@
 ## Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 
 import cups, pprint, os, tempfile, re
-from rhpl.translate import _, N_
+from gettext import gettext as _
 import locale
 from debug import *
 
